xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-rest/src/test/java/org/xwiki/livedata/internal/rest/DefaultLiveDataEntriesResourceTest.java

```diff
@@ -179,7 +181,12 @@ void getEntriesWithNamespaceNull() throws Exception
         assertEquals("{\"query\":{"
             + "\"properties\":[\"prop1\",\"prop2\"],"
             + "\"source\":{\"id\":\"sourceId\"},"
-            + "\"filters\":[{\"property\":\"age\",\"constraints\":[{\"value\":\"18\"}]}],"
+            + "\"filters\":["
+            + "{\"property\":\"age\"},"
```

Comment:
Maybe we should drop a filter if it doesn't have any constraints.

Reply:
Done.